fastlane-config/extract_config.rb

```
@@ -0,0 +1,60 @@
+#!/usr/bin/env ruby
```

RuboCop (1.84.1)

[warning] Script file extract_config.rb doesn't have execute permission.

(Lint/ScriptPermission)